common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOtherCache.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.identity.common.internal.cache;
+
+import android.content.Context;
+import androidx.annotation.VisibleForTesting;
+import com.microsoft.identity.common.adal.internal.cache.IStorageHelper;
+import com.microsoft.identity.common.adal.internal.cache.StorageHelper;
+
+public class MsalOtherCache {
+
+    /**
+     * The name of the SharedPreferences file on disk.
+     * This file is only used to store MSAL CPP authority validation metadata
+     */
+    public static final String DEFAULT_CPP_AUTHORITY_VALIDATION_METADATA_SHARED_PREFERENCES =
+            "com.microsoft.identity.client.cpp_authority_validation_metadata";
+
+    private final ICacheKeyValueDelegate mCacheKeyValueDelegate;
+
+    private final ISharedPreferencesFileManager mCppAuthorityValidationMetadataSharedPreferencesFileManager;
+
+    public MsalOtherCache(final Context context)
+    {

```

Comment:
As a minor style-nit.... 
```java
public void weUsuallyDoThis() {
  // hello!
}
```
instead of...
```java
public void puttingOurBracesLikeThis()
{
   // goodbye!
}
```

Reply:
Here and in other parts of this file/PR